packages/loot-core/src/shared/util.ts

```diff
@@ -329,6 +329,21 @@ export function getNumberFormat({
 
 // Number utilities
 
+/**
+ * The exact amount.
+ */
+export type Amount = number;
```

Comment:
Curious about your thoughts on making these [branded types](https://stackoverflow.com/a/61621026) so they are not interchangeable? Otherwise correct me if I'm wrong but I think TypeScript would allow substituting `IntegerAmount` for `Amount`

Reply:
I tried implementing it briefly and it results in a lot of typecheck errors:
![image](https://github.com/user-attachments/assets/5ec19ca8-8498-4327-99bc-99100dc870b9)

I think it's a good idea and we should implement it, but we do it in a separate PR. WDYT?

Reply:
Yeah sounds good